server/src/test/java/com/objectcomputing/checkins/services/TestContainersSuite.java

```diff
@@ -32,6 +32,46 @@ public abstract class TestContainersSuite implements RepositoryFixture, TestProp
 
     public TestContainersSuite() {}
 
+    @BeforeEach
```

Comment:
Nothing changed other than the annotations being added back. The diff just shows this because I moved the deleteAllEntities method to the bottom of the class